docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -31,6 +31,31 @@ command:
 
     openssl rand 512 | base64
 
+.. _verify_secrets:
+
+Verify configuration
+^^^^^^^^^^^^^^^^^^^^
+
+To verify shared secret configuration:
+
+1. Start your Trino cluster with two or more nodes configured with a shared
+   secret.
+2. Connect to the :doc:`Web UI </admin/web-interface>` from your browser using a
+   URL that uses HTTPS, such as ``https://trino.example.com:8443``. Enter any
+   username into the ``Username``
+   text box, and log in to the UI. The ``Password`` box is disabled while
+   :doc:`authentication <authentication-types>` is not configured.
+3. Confirm the number of ``ACTIVE WORKERS`` equals the number of nodes
+   configured with your shared secret.
+4. Stop your Trino cluster, change the value of the shared secret on one node,
+   and restart your cluster.
+5. Log in to the Web UI and confirm the number of ``ACTIVE WORKERS`` is one
+   fewer, and therefore not authenticated.
```

Comment:
one less, and the worker with the invalid secret is not authenticated and therefore not registered with the coordinator.